util/cron/load-base-deps.bash

```diff
@@ -2,6 +2,13 @@
 
 # Sources base dependencies required for all tests.
 
+# Prevent loading more than once
+if [ "$CHPL_BASE_DEPS_LOADED" = "true" ]; then
+  echo "Base dependencies already loaded, exiting $0 without reloading them..."
```

Comment:
$0 --> ether `$BASH_SOURCE` or `${BASH_SOURCE[0]}` please

Note that I am making this change elsewhere in #26298.